recipes/Avocado-Toast.qmd

```diff
@@ -0,0 +1,24 @@
+---
+  title: "Template for new recipe"
+image: "https://i2.wp.com/earthblokes.com/wp-content/uploads/2021/02/vegan-avocado-toast-4.jpg?resize=800%2C1200&ssl=1"
+image-alt: image-alt: "A delicious avocado toast"
+---
+
+![A delicious avocado toast](https://i2.wp.com/earthblokes.com/wp-content/uploads/2021/02/vegan-avocado-toast-4.jpg?resize=800%2C1200&ssl=1)
+
+categories: - \["Breakfast", "Snack"\] ---
+
+\## {{< fa basket-shopping >}} Ingredients
+
+-   ingredients
+-   [ ] 1 ripe avocado
+-   [ ] 2 slices of bread
+-   [ ] Salt to taste
+-   [ ] Pepper to taste
+-   [ ] Lemon juice
+
+## {{< fa kitchen-set >}} Procedure
+
+1.  Step 1 Toast the slices of bread to your desired level of crispiness.
+2.  Step 2 Cut the avocado in half, remove the pit, and scoop the flesh into a bowl. Mash the avocado with a fork until smooth.
+3.  Step 3 Spread the mashed avocado evenly onto the toasted bread slices. Season with salt and pepper to taste. Add a squeeze of lemon juice if desired for extra flavor.
```

Comment:
Please remove "Step 1", "Step 2" and "Step 3" in the list. The numbers will take care of enumeration.